index.html

```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
             </div>
         </section>
         <!-- end of about -->
-        <!-- <section  class="get-in-touch">
-            <h1>Get in</h1>
-        </section> -->
+        <section  class="get-in-touch">
+            <h3 hidden>heading</h3>
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="/" method ="post">
```

Comment:
- [ ] As requested by the previous reviewer, Please add action value to your form so it should be submitted to [formspree service](https://formspree.io/html) when users click on the submit button. Action value should be similar to `https://formspree.io/f/{form_id}.` Create `form_ID` on formspree.io.